res2df/equil.py

```diff
@@ -132,78 +132,78 @@ def df(
+            be inferred if not present in :term:`deck`
     """
     if "EQLDIMS" not in deck:
         deck = inferdims.inject_xxxdims_ntxxx("EQLDIMS", "NTEQUL", deck, ntequl)
-    return common.ecl_keyworddata_to_df(
+    return common.keyworddata_to_df(
         deck, "RSVD", renamer=RENAMERS["RSVD"], recordcountername="EQLNUM"
     )
 
 
 def rvvd_fromdeck(
     deck: Union[str, "opm.libopmcommon_python.Deck"], ntequl: Optional[int] = None
 ) -> pd.DataFrame:
-    """Extract RVVD data from a deck
+    """Extract RVVD data from a :term:`deck`
 
     Args:
         deck
-        ntequl: Number of EQLNUM regions in deck. Will
-            be inferred if not present in deck
+        ntequl: Number of EQLNUM regions in :term:`deck`. Will
+            be inferred if not present in :term:`deck`
     """
     if "EQLDIMS" not in deck:
         deck = inferdims.inject_xxxdims_ntxxx("EQLDIMS", "NTEQUL", deck, ntequl)
-    return common.ecl_keyworddata_to_df(
+    return common.keyworddata_to_df(
         deck, "RVVD", renamer=RENAMERS["RVVD"], recordcountername="EQLNUM"
     )
 
 
 def pbvd_fromdeck(
     deck: Union[str, "opm.libopmcommon_python.Deck"], ntequl: Optional[int] = None
 ) -> pd.DataFrame:
-    """Extract PBVD data from a deck
+    """Extract PBVD data from a :term:`deck`
 
     Args:
         deck
-        ntequl: Number of EQLNUM regions in deck. Will
-            be inferred if not present in deck
+        ntequl: Number of EQLNUM regions in :term:`deck`. Will
+            be inferred if not present in :term:`deck`
     """
     if "EQLDIMS" not in deck:
         deck = inferdims.inject_xxxdims_ntxxx("EQLDIMS", "NTEQUL", deck, ntequl)
-    return common.ecl_keyworddata_to_df(
+    return common.keyworddata_to_df(
         deck, "PBVD", renamer=RENAMERS["PBVD"], recordcountername="EQLNUM"
     )
 
 
 def pdvd_fromdeck(
     deck: Union[str, "opm.libopmcommon_python.Deck"], ntequl: Optional[int] = None
 ) -> pd.DataFrame:
-    """Extract PDVD data from a deck
+    """Extract PDVD data from a :term:`deck`
 
     Args:
         deck
-        ntequl: Number of EQLNUM regions in deck. Will
-            be inferred if not present in deck
+        ntequl: Number of EQLNUM regions in :term:`deck`. Will
+            be inferred if not present in :term:`deck`
     """
     if "EQLDIMS" not in deck:
         deck = inferdims.inject_xxxdims_ntxxx("EQLDIMS", "NTEQUL", deck, ntequl)
-    return common.ecl_keyworddata_to_df(
+    return common.keyworddata_to_df(
         deck, "PDVD", renamer=RENAMERS["PDVD"], recordcountername="EQLNUM"
     )
 
 
 def phases_from_deck(deck: Union[str, "opm.libopmcommon_python.Deck"]) -> str:
-    """Determined the set of phases from a deck, as
+    """Determined the set of phases from a :term:`deck`, as
     a string with values "oil-water-gas", "gas-water", "oil-water",
     or "oil-gas"
 
     Args:
-        deck: A parsed deck or DATA-file as a string
+        deck: A parsed :term:`deck` or DATA-file as a string
```

Comment:
DATA-file could link to the glossary here also. Should be consistent and call it `.DATA file`.